modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -59,7 +59,6 @@ interface ViewObjectDefinitionsProps extends IFDSTableProps {
 	learnResourceContext: any;
 	modelBuilderURL: string;
 	nameMaxLength: string;
-	objectDefinitionsAPIURL: any;
 	objectDefinitionsCreationMenu: {
```

Comment:
Hey @p-albuquerque, since you will no longer need this property, it would no longer need to be passed through the `view_object_definitions.jsp`. So you can remove the snippet:

```
.put(
    "objectDefinitionsAPIURL", viewObjectDefinitionsDisplayContext.getAPIURL()
)
```